Fleshes out the summary and use case language. (#1)
SteveLasker/draft-lasker-vcon-lifecycle: README.md

# Introduction

> [vCons][draft-vcon] are a powerful means to capture and collaborate on the details of human coversations, feeding AI systems, enabling entities to more accurately respond to customer needs, and to more effectively manage the health of their business. The vCon working group is about passing conversational data, such data commonly generated and collected in business and security environments, from chat logs to transcripts to recordings.  Most systems provide a way to store such information, but there are not many standards or interoperability within the storage or transmission mechanisms. [vcon] is a framework for capturing and collaborating on the details of a Virtual Conversation, feeding AI systems, enabling entities to more accurately respond to customer needs, and entities to more effectively manage the health of their business.   The two opposing forces influencing such information passing are trying to enforce privacy of personal data and providing the ability and interest to use conversations in various ways, e.g.,  AI analysis.  

Reviewer: ```suggestion
[vCons][draft-vcon] are powerful means of capturing and collaborating on the details of human conversations.
They feed AI systems, enable entities to respond more accurately to customer needs, and manage the health of their business more effectively.
The vCon working group is about passing conversational data, such as data commonly generated and collected in business and security environments, from chat logs to transcripts to recordings. 
Most systems provide a way to store such information, but there are few standards or interoperability within the storage or transmission mechanisms.
[vcon] is a framework for capturing and collaborating on the details of a Virtual Conversation, feeding AI systems, and enabling entities to respond to customer needs more accurately and manage their business's health more effectively.
The two opposing forces influencing such information passing are trying to enforce personal data privacy and providing the ability and interest to use conversations in various ways, e.g.,  AI analysis.  
```

> Fundamental to the responsible protection of data and the ability to leverage that data is the proper management of consent. Consent granted by the data subject of a vCon also needs to be stored and passed, exactly like the other information contained in a vCon.  Unlike the rest of the information contained, the gathered consent is expressly not immutable.  Consent, when gathered for a purpose, can also be revoked by the data subject, at any time, and surely after a vCon has been shared for analysis. Multiple regulations, including the US born [California Consumer Protection Act (CCPA)][CCPA] and the EU born [General Data Protection Regulation (GDPR)][GDPR] outline requirements for entities to responsibly use and dispose of PII information upon request. Consent, although illustrative, is not the only example of mutability in the lifecycle of a vCon. Verification of parties, improvements in the analsys and additions of contextual attachments result in updates to a vCon after data is shared, begging for a mechanism to track and govern such changes. 

Reviewer: ```suggestion
Proper consent management is fundamental to the responsible protection of data and the ability to leverage that data.
Consent granted by the data subject of a vCon also needs to be stored and passed, exactly like the other information contained in a vCon.
Unlike the rest of the information contained, the gathered consent is expressly not immutable.
Consent, when gathered for a purpose, can also be revoked by the data subject at any time and surely after a vCon has been shared for analysis.
Multiple regulations, including the US-born [California Consumer Protection Act (CCPA)][CCPA] and the EU-born [General Data Protection Regulation (GDPR)][GDPR], outline requirements for entities to use and dispose of PII information responsibly upon request.
Consent, although illustrative, is not the only example of mutability in the lifecycle of a vCon.
Verification of parties, improvements in the analysis, and additions of contextual attachments result in updates to a vCon after data is shared, begging for a mechanism to track and govern such changes.
```

> More so, the vCon is properly understood within the context of consent.  That is, the times and places vCons are created are vast, from consumer facing organizations, regulated industries and security applications. In each case, these conversations are collected under regulation and legal compliance. The requirement to prove and assure the presence of the proper governance of the conversation is generally assumed by those that use vCons; other uses are considered out of scope.  That is, the fundamental goal of vCons is to provide a definition of a conversation precisely so that third party tools and organizations can properly protect and support the digital rights of those recorded. In plain terms, vCons are a tool to enable good actors to "do the right thing" and to prove it, at scale.  Except for the implications their absence may imply, vCons do not detect actors unconcerned with the proper protection of the personal data.   

Reviewer: ```suggestion
Moreover, the vCon is properly understood within the context of consent.
That is, the times and places vCons are created are vast, ranging from consumer-facing organizations to regulated industries and security applications.
In each case, these conversations are collected under regulation and legal compliance.
Those who use vCons generally assume the requirement to prove and assure the proper governance of the conversation; other uses are considered out of scope.
The fundamental goal of vCons is to define a conversation precisely so that third-party tools and organizations can properly protect and support the digital rights of those recorded.
In plain terms, vCons are a tool to enable good actors to "do the right thing" and prove it at scale.
Except for the implications their absence may imply, vCons do not detect actors unconcerned with the proper protection of personal data.   
```

> Although vCons are a tool to enable actors to do the right thing, it is not a tool that enables actors in a distributed system to prove it.  However, when combined with the SCITT protocol, proof becomes practical. SCITT provides Relying Parties to obtain information pertinent to the lifecycle of a vCon in a way that is "transparent".  SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.  These relying parties can then use this information to make decisions based on the current state and context of the vCon, to account for changes in consent, updates in the accuracy of the content or ammendendment of the information it might contain. SCITT, the supply chain, integrity and trust protocol, enables clients to register statements about events, physical or virtual, such as when things are created or used.  These statements are immutable, and are useful to support auditing, governance and coordination between various distributed systems. 

Reviewer: ```suggestion
Although vCons are a tools that enable actors to do the right thing, they are not tools that enable actors in a distributed system to prove it.
However, when combined with the SCITT protocol, proof becomes practical.
SCITT allows Relying Parties to obtain information pertinent to the lifecycle of a vCon in a "transparent" way.
SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.
These relying parties can then use this information to make decisions based on the current state and context of the vCon to account for changes in consent, updates in the accuracy of the content, or amendments of the information it might contain.
SCITT, the Supply Chain, Integrity, Transparency, and Trust protocol, enables clients to register statements about events, physical or virtual, such as when things are created or used.
These statements are immutable and are useful to support auditing, governance, and coordination between various distributed systems. 
```

> This draft provides an overview of the requirements and a workflow to onboard customers with end-to-end, interoperable services and tooling which honors the charter of the working group to safely pass conversational data between consenting parties.  The workflow enables entities in the workflow to be collaborate on a vCon, while assuring all entities in the workflow are adhering to relevant PII regulations using a [Supply Chain, Integrity, Transparency and Trust (SCITT)][draft-scitt] Ledger.  Actual implementations of these workflows are left to implementers and applications; this draft provides an authoritative list of the entries that should appear on the SCITT transparency ledger to enable them. 

Reviewer: ```suggestion
This draft provides an overview of the requirements and a workflow for onboarding customers with end-to-end, interoperable services and tooling, which honors the working group's charter to pass conversational data safely between consenting parties.
The workflow enables entities in the workflow to collaborate on a vCon while assuring all entities in the workflow adhere to relevant PII regulations using a [Supply Chain, Integrity, Transparency and Trust (SCITT)][draft-scitt] Ledger.
Actual implementations of these workflows are left to implementers and applications; this draft provides an authoritative list of the entries that should appear on the SCITT transparency ledger to enable them. 
```

- **SCITT Transparency Service**: An append-only ledger for integrity protecting vCons, ensuring the state of a vCon is known at a point in time for conformance to governance and regulatory requirements.
- **vCon Registry**: A storage service, capable of storing vCons, including rich metadata and larger attachments including audio and video recordings.

# Example Use Cases

## vCon Lifecycle: Creation, Distribution and Deletion

> The vCon lifecycle is a series of phases that occur in a vCon, from creation, through distribution to deletion.  The tracking of these phases aids in enabling of fundamental privacy rights such as the right to know how your data was processed ("Right to Know") and secures AI supply chains by establishing provenance and guaranteeing integrity.  

Reviewer: ```suggestion
The vCon lifecycle is a series of phases that occur in the life of a vCon, from creation through distribution to deletion.
Tracking these phases aids in enabling fundamental privacy rights, such as the right to know how your data was processed ("Right to Know"), and secures AI supply chains by establishing provenance and guaranteeing integrity.  
```

The phases of a vCons life include: 

1. **vCon Created**: The call completes and a vCon is created with call metadata stored in the vCon Registry.

2. **Recording Added**: The actual recording is saved and added to the attachments section of the vCon.

3. **vCon Sent**: The Data Originator sends the vCon to the Data Controller with integrity protection using SCITT.

4. **vCon Received**: The Data Controller receives the vCon and records this in the SCITT Transparency Service.

5. **vCon Enhanced**: The Data Controller adds transcription, license information, and identifies themselves.

6. **vCon Sent to Processors**: The Data Controller sends the vCon to relevant Data Processors.

> 7. **Data Processing**: Value-added services are performed on the vCon data.

Reviewer: ```suggestion
1. **vConCreated**: The call completes, and a vCon is created with call metadata stored in the vCon Registry.
2. **RecordingAdded**: The actual recording is saved and added to the attachments section of the vCon.
3. **vConSent**: The Data Originator sends the vCon to the Data Controller with integrity protection using SCITT.
4. **vConReceived**: The Data Controller receives the vCon and records this in the SCITT Transparency Service.
5. **vConEnhanced**: The Data Controller adds transcription and license information and identifies themselves.
6. **vConSentToProcessors**: The Data Controller sends the vCon to relevant Data Processors.
7. **Data Processing**: Value-added services are performed on the vCon data.
```

> 
> 8. **vCon Deletion**: The vCon is deleted when no longer needed, when consent is revoked, or when it expires.
> 
> 

Reviewer: ```suggestion
8. **vConDeletion**: The vCon is deleted when no longer needed, when consent is revoked, or when it expires.
```

## Digital Rights Management

> Interwoven with the vCon lifecycle is the management of consent that applies to the vCon.  In this case, a more modern idea of consent is imagined: consent is gathered by the data controller, for a particular purpose such a training or sharing, that can be withdrawn by the data subject on demand.  This withdrawl of consent may result in the revocation of a vCon, or in it's modification to remove the non-consenting portions. 

Reviewer: ```suggestion
Interwoven with the vCon lifecycle is the management of consent that applies to the vCon.
In this case, a more modern idea of consent is imagined: consent is gathered by the data controller, for a particular purpose such a training or sharing, that can be withdrawn by the data subject on demand.
This withdrawl of consent may result in the revocation of a vCon, or in it's modification to remove the non-consenting portions. 
```

> Lifecycle events in Digital Rights Management include: 
> 
> 1. **Consent to Record**: Consent is requested from the Data Subject to record the conversation.
> 
> 2. **Consent for Purpose**: Confirmation of consent for specific purposes (e.g., "sales followup") is obtained.
> 
> 3. **Consent Recorded**: The Data Controller records where consent was confirmed in the transcript or recording.
> 
> 4. **Consent Receipt Sent**: Notification is sent to Data Subject(s) with a link to review consent details.
> 
> 5. **Consent Review/Revocation**: Data Subject can review or choose to revoke consent at any time.

Reviewer: ```suggestion
Lifecycle events in Digital Rights Management include: 

1. **ConsentToRecord**: Consent is requested from the Data Subject to record the conversation.
2. **ConsentForPurpose**: Confirmation of consent for specific purposes (e.g., "sales followup") is obtained.
3. **ConsentRecorded**: The Data Controller records where consent was confirmed in the transcript or recording.
4. **ConsentReceipt Sent**: Notification is sent to Data Subject(s) with a link to review consent details.
5. **ConsentReview-Revocation**: Data Subject can review or choose to revoke consent at any time.
```

> 6. **Revoke Request Processing**: If consent is revoked, the request is processed and communicated to all parties.

Reviewer: ```suggestion
6. **RevokeRequestProcessing**: If consent is revoked, the request is processed and communicated to all parties.
```

## Ammendment of Existing vCons

> Under normal circumstances, vCons may be ammended. For example, at the time of creation, the parties to a vCon may be verified by a number of existing methods such as OAuth.  However, in the case were account credentials are found to be lost or compromised, this opnion may be revisited.  Issues with Party Verification are a prime cause of
> "Rights to Correct" requests, fundamental to responsible data rights management.  Other enchancement and modification of vCons may occur for many other security reasons, future processing needs or in response to changes in regulations or compliance requirements. 
> 
> Events that may be recorded on the distributed ledger include:
> 
> 1. **Dialog Added**: A dialog is saved and added to the vCon.
> 
> 2. **vCon Processed**: The Data Controller may process the vCon 
> 
> 3. **Data Redaction**: Data Processors delete the data or redact the Data Subject.

Reviewer: ```suggestion
Under normal circumstances, vCons may be amended.
For example, at the time of creation, the parties to a vCon may be verified by a number of existing methods such as OAuth.
However, in the case where account credentials are found to be lost or compromised, the "opinion of being verified" may be revisited.
Issues with Party Verification are a prime cause of "Rights to Correct" requests, and they are fundamental to responsible data rights management.
Other enhancements and modifications of vCons may occur for many other security reasons, future processing needs, or in response to changes in regulations or compliance requirements. 

Events that may be recorded on the distributed ledger include:

1. **DialogAdded**: A dialog is saved and added to the vCon.
2. **vConProcessed**: The Data Controller may process the vCon.
3. **Data Redaction**: Data Processors delete the data or redact the Data Subject.
```

[GDPR]:                 https://gdpr.eu/
[hippa]:                https://www.hhs.gov/hipaa/index.html
[privacy-primer-vcon]:  https://datatracker.ietf.org/doc/draft-james-privacy-primer-vcon/

# vCon Lifecycle Events and Operations

> vCons contain multiple elements of Personally Identifiable Information of the parties involved, including phone numbers, emails, voice & facial prints and location information. These elements are what make vCons so powerful to assist customers.
> However, the information must be handled responsibly, giving the involved parties an opportunity to consent to their information being captured, and the ability to revoke consent with the right to be forgotten.
> 
> 
> 

Reviewer: ```suggestion
vCons contain multiple elements of Personally Identifiable Information of the parties involved, including phone numbers, emails, voice & facial prints, and location information.
These elements are what make vCons so powerful in assisting customers.
However, the information must be handled responsibly, allowing the involved parties to consent to their data being captured and the ability to revoke consent with the right to be forgotten.
```
